lib.go

```diff
@@ -52,6 +53,30 @@ func (f *File) saveFileList(name string, content []byte) {
 	f.XLSX[name] = newContent
 }
 
+// writeXMLToZipWriter writes data obj in form of XML to zip,Writer
+func writeXMLToZipWriter(zw *zip.Writer, name string, data interface{}) error {
+	w, err := zw.Create(name)
+	if err != nil {
+		return err
+	}
+	_, err = w.Write([]byte(XMLHeader))
+	if err != nil {
+		return err
+	}
+	encoder := xml.NewEncoder(w)
+	return encoder.Encode(data)
+}
+
+// writeStringToZipWriter writes string to zip,Writer
```

Comment:
Extra comma in the comment here

Reply:
checking grammer in comment :p

Reply:
I know it's a little trivial but clean comments help everyone who uses the code

Reply:
comment updated